File: core-java-practice/leet-code-codebase/MoveZeroes.java
public class MoveZeroes {
    public static void movezeroes(int[] num){
          int j=0; //position 

          for(int i=0;i<num.length;i++){
            if(num[i]!=0){
                num[j++]=num[i]; //move non zero
            }
            while(j<num.length){
                num[j++]=0; //fill the zeroes
            }
          }

    }
    
}

File: core-csharp-practice/leet-code-codebase/MoveZeroes.cs
using System;

class MoveZeroes{
    static void func(int[] num){
        int j=0; //for position

        for(int i=0;i<num.Length;i++){
            if(num[i]!=0){
                num[j++]=num[i];  //move non zero
            }
        }
            while(j<num.Length){
                num[j++]=0;      //fill the zero in last 
            }
    }
}